src/components/landingPageDuplicate.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from "react";
+import logotest from "../MYtineraryLogo.png";
+import cities from "../circled-right-2.png";
+import "../App.css";
+import { NavLink } from "react-router-dom";
+import Header from "./header";
+import Slider from "react-animated-slider";
+import "react-animated-slider/build/horizontal.css";
+import Amsterdam from "../amsterdam.jpg";
```

Comment:
To make this a little more extendable/manageable, I would put all the images into a JSON file and then reference those sources as how they are served by the webpack server.  For instance:
```
[
  {
    "name": "Amsterdam",
    "source": "images/amsterdam.jpg"
  }
]
```